docs/internals/layout_in_storage.rst

```diff
@@ -6,32 +6,43 @@ Layout of State Variables in Storage
 
 .. _storage-inplace-encoding:
 
-Statically-sized variables (everything except mapping and dynamically-sized
-array types) are laid out contiguously in storage starting from position ``0``.
+State variables of contracts are stored in storage in a compact way such
+that multiple values sometimes use the same storage slot.
+Except for dynamically-sized arrays and mappings (see below), data is stored
+contiguously item after item starting with the first state variable,
+which is stored in slot ``0``. For each variable,
+a size in bytes is determined according to its type.
 Multiple, contiguous items that need less than 32 bytes are packed into a single
```

Comment:
That's actually not what I meant (two times 16 would also be fine).

Reply:
Oh, maybe I misunderstood this? I thought this sentence says that if you have several items, the "groups" that together take 32 bytes or less get packed into a single slot (with additional rules clarfied below).

Or do you just mean that there should also be `no more than` instead of `less than`?